sklearn/linear_model/logistic.py

```diff
@@ -1286,7 +1286,7 @@ class LogisticRegression(BaseEstimator, LinearClassifierMixin,
         - 'newton-cg', 'lbfgs', 'sag' and 'saga' handle L2 or no penalty
         - 'liblinear' and 'saga' also handle L1 penalty
         - 'saga' also supports 'elasticnet' penalty
-        - 'liblinear' does not handle no penalty
+        - 'liblinear' does not support setting `penalty=None`
```

Comment:
should be penalty='none', please use double backticks instead of single backticks.